util/dvsim/FlowCfg.py

```diff
@@ -161,6 +157,13 @@ def resolve_hjson_raw(self, hjson_dict):
                 self_val = getattr(self, key)
                 scalar_types = {str: [""], int: [0, -1], bool: [False]}
 
+                # Case 0: We don't yet have a proper value for this key (so
+                # self_val is None). Take the value in the config file.
+                if self_val is None:
+                    setattr(self, key, hjson_dict_val)
+                    rm_hjson_dict_keys.append(key)
+                    continue
```

Comment:
Would you mind removing this and updating the clause on 168 to `elif` instead? The case you added IMO is related to the the conditionals below.

Reply:
Ok, done.